frontend/helpers/algolia-helpers.ts

```diff
@@ -23,3 +23,10 @@ export function formatFacetName(algoliaName: string): string {
    }
    return FACETS_NAME_OVERRIDES[algoliaName];
 }
+
+/**
+ * https://www.algolia.com/doc/api-reference/api-parameters/filters/#usage-notes
+ */
+export function escapeFilterValue(value: string) {
+   return value.replaceAll("'", "\\'").replaceAll('"', '\\"');
```

Comment:
Even though their docs are interactive and provide code examples, it's still unclear what the *spec* is. After interpreting their code examples and interacting with the validator I read the spec as follows.

> All single quotes and double quotes should be preceded with a backslash: `\`

In most languages, this requires two backslashes when used in a string literal which is the case here. I wrote this mostly to make sure my understanding is correct.

Reply:
I understand it the same way you do. I think Algolia's client wants to receive a string with with every `'` and `"` preceded by `\`.